js/redirects.js

```diff
@@ -26,6 +38,19 @@ $( document ).ready(function() {
         "to": "http://kubernetes.io/docs/whatisk8s/"
     }];
 
+    for (var i = 0; i < apiv1.length; i++) {
+        if (forwardingURL.indexOf(apiv1[i].from) > -1) {
+            var re = new RegExp(apiv1[i].pattern, 'g');
+            var matchary = re.exec(forwardingURL);
+            var newURL = apiv1[i].to;
+            if (matchary !== null) {
+                newURL += apiv1[i].postfix.replace("<token>", matchary[1]);
+            }
+            notHere = true;
+            window.location.replace(newURL);
+        }
+    }
+
```

Comment:
As long as we have an issue tracking it, we can take care of that at a later time. Since there's 1-2 people logging issues unable to find kubectl documentation, this fix is probably higher priority.